pandas/core/series.py

```diff
@@ -4736,12 +4737,24 @@ def between(self, left, right, inclusive=True) -> Series:
         3    False
         dtype: bool
         """
-        if inclusive:
+
+        if inclusive is True or inclusive == "both":
             lmask = self >= left
             rmask = self <= right
-        else:
+        elif inclusive == "left":
+            lmask = self >= left
+            rmask = self < right
+        elif inclusive == "right":
+            lmask = self > left
+            rmask = self <= right
+        elif inclusive is False or inclusive == "neither":
             lmask = self > left
             rmask = self < right
+        else:
+            raise ValueError(
+                "Input should be boolean or string of 'both','left', 'right', "
+                "or 'neither'"
```

Comment:
Should we exclude the boolean if it is being silently hidden? Also this needs tests for code coverage of the raised error.

Reply:
That's a good question about hiding the boolean, right now I have it so that the standard string values are suggested first and then the boolean is mentioned. Do you think it would be best to exclude the boolean being the arrangement that we have right now?
We're working on getting more documented tests for code coverage over these next few days and should have wider coverage for our additions soon.